lib/config.go

```diff
@@ -6,9 +6,13 @@ import (
 	"os"
 )
 
-func getConfigPath() string {
-  dirname, _ := os.UserHomeDir()
-  return dirname + "/.mdrss/config.json"
+func getConfigPath(config_path string) string {
+  if FileExists(config_path) {
+    return config_path
+  } else {
+    dirname, _ := os.UserHomeDir()
+    return dirname + "/.mdrss/config.json"
+  }
```

Comment:
I check for the existence of the config path later in the code. Here I just select / construct the config path that I want to use. Hence. the first if statement should **only** check whether the config path from the flag is enabled. 

(PS: it should not be called configPath yet, because we don't know whether it is even used at this point. It's really the config path **flag**).

Reply:
I'd also add that XDG should be used. So this should be `~/.config/mdrss/config.json`.

Instead of:

```go
dirname, _ := os.UserHomeDir()
return dirname + "/.mdrss/config.json"
```
    
you can use [adrg/xdg](https://github.com/adrg/xdg):

```go
configFilePath, err := xdg.ConfigFile("mdrss/config.json")
if err != nil {
 log.Fatal(err)
}
return configFilePath
```